src/app/f2f/controllers/checkDetails.js

```diff
@@ -167,8 +168,9 @@ class CheckDetailsController extends DateController {
 
       await this.saveF2fData(req.axios, f2fData, req);
       callback();
-    } catch (err) {
-      callback(err);
+    } catch (error) {
+      console.log("Error", error);
```

Comment:
Need to remove?

Reply:
I added it to help debugging when there is an error but I can remove if we don't think it's helpful?

Reply:
Removed